less/index.less

```diff
@@ -1,9 +1,39 @@
 // Is this working?
-/* http://meyerweb.com/eric/tools/css/reset/ 
+/* http://meyerweb.com/eric/tools/css/reset/
    v2.0 | 20110126
    License: none (public domain)
 */
 
+//flexbox mxin
+.flexy(
+  @disp: flex,
+  @dir: null,
+  @wrap: null,
+  @just: null,
+	@ai: null) {
```

Comment:
Be careful with the `null` keyword - this works with Sass (since it omits nulled properties when compiling) but not with Less. Perhaps put in the default values for these properties instead?